www/src/models/transformation/TransformationResult.ts

```diff
@@ -0,0 +1,8 @@
+import { Record } from "gena-app";
+
+export interface TransformationTable extends Record<number> {
```

Comment:
I thought we want to change the name of this?

Reply:
updated it as TransformationResult